javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,55 @@
+//Excercise 1: ageify
+function ageify(yearOfBirth, yearFuture) {
+    let age = yearFuture - yearOfBirth;
+    return "You will be " + age + " years old in " + yearFuture;
+}
+
+ageify(1992, 2099);
+
+
+//Excercise 2: dog years
+
+function dogAge(dogYearOfBirth, dogYearFuture, shouldResultInDogYears) {
+    let dogYear = dogYearFuture - dogYearOfBirth;
+    if (shouldResultInDogYears === false) {        
+        var humanOrDog = " human years old in "
+    } else {
+        dogYear *= 7;
+        var humanOrDog = " dog years old in "
+    }
+
+    return "Your dog will be " + dogYear + humanOrDog + dogYearFuture;
+
+}
+
+dogAge(2015, 2025, true);
```

Comment:
check it again
delete following statements 
var humanOrDog = " human years old in "
var humanOrDog = " dog years old in "
use console.log() instead in both places. In which you must show calculated age.

use _let_ or _const__ instead of _var_
delete return as well.